### DISCUSSION TITLE
Possible to tweak enemy hideouts/ ammo cache

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1544] [Question and answer] [answered]
I'm planning to use custom compositions for hideouts and cache, is it possible?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Depend on your skill here is:
- https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/hideout/create_composition.sqf
- https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L211-L225
- https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L196-L209
- https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/cache/create.sqf#L51-L68

Cheers
